tikv-configuration-file.md

```diff
@@ -1933,6 +1933,20 @@ Configuration items related to Raft Engine.
 + If there are multiple disks on your machine, it is recommended to store the data of Raft Engine on a different disk to improve TiKV performance.
 + Default value: `""`
 
+### `spill-dir` <span class="version-mark">New in v8.4.0</span>
+
++ The auxiliary directory for storing raft log files. When the disk where the `dir` specified is full, new raft logs will be stored under this directory. If this directory does not exist after configuration, it will be created when TiKV is started.
++ If this configuration is not set, it means that the auxiliary directory is disabled.
```

Comment:
```suggestion
+ The auxiliary directory for storing Raft log files. When the disk for the `dir` directory is full, new Raft logs will be stored under this directory. If this auxiliary directory does not exist after configuration, it will be automatically created when TiKV is started.
+ If this configuration is not set, the auxiliary directory is not enabled.
```